src/components/IntegrationTest/tabs/IntegrationTestOverviewTab.tsx

```diff
@@ -124,6 +134,36 @@ const IntegrationTestOverviewTab: React.FC<IntegrationTestOverviewTabProps> = ({
                   })}
                 </>
               )}
+              {params && (
+                <DescriptionListGroup data-test="its-overview-params">
+                  <DescriptionListTerm>
+                    Parameters{' '}
+                    <Tooltip content="Parameters which will be added to the integration Tekton pipeline run.">
+                      <OutlinedQuestionCircleIcon />
+                    </Tooltip>
+                  </DescriptionListTerm>
+                  <DescriptionListDescription>
+                    {`${params.length} parameters`}
```

Comment:
use pluralized string only when the length is greater than `1`.
![image](https://github.com/openshift/hac-dev/assets/9964343/64c25724-1264-44df-95c2-8b8978487f51)